packages/plugin-simple-throttle/test/throttle.test.ts

```diff
@@ -3,11 +3,21 @@ import plugin from '../'
 import Client from '@bugsnag/core/client'
 
 describe('plugin: throttle', () => {
-  it('prevents more than maxEvents being sent', () => {
+  describe('addOnError', () => {
```

Comment:
I don't think we need this second describe

Reply:
agreed. not a lot of functionality here to describe separately